### DISCUSSION TITLE
Sensori Aquara non rilevati da Xiaomi Home

### DISCUSSION THREAD

**Discussion by Vinz1985** [indomus/forum#429] [HUB: Home Assistant]
Ciao ragazzi, non so se è la sezione giusta, in caso contrario spostate pure il post. Arrivo al dunque:
ho acquistato sensori vari Aquara e li ho subito connessi al gateway (conbee II) e importati tramite Deconz su HA, tutto perfettamente riuscito al primo colpo ma non riesco a farmeli riconoscere da Alexa, in realta non riesco proprio a farmeli riconoscere dall'app Xiaomi Home in quanto, in fase di accoppiamento, mi dice che non ho il Gateway Aquara e dunque se non riesco a portarli sull'app non riesco nemmeno ad associarli tramite skill ad Alexa.
Qualcuno di voi è riuscito, senza possedere lo stesso gateway dei sensori, a farli riconoscere dall'app Xiaomi?
Grazie...

**Comment by indomus** [comment]
E' normale: non puoi accoppiare un componente ZigBee a più di un Bridge alla volta.

**Reply by Vinz1985** [reply]
In che senso? Scusa io vorrei semplicemente che l'app Xiaomi mi riconoscesse il mio gateway Conbee II mentre vuole solo l'hub Xiaomi Aquara. 
Ti faccio un esempio se avessi avuto l'hub Aquara, l'app Xiaomi mi avrebbe riconosciuto automaticamente l'hub e di conseguenza i device. 
Secondo la tua risposta i sensori o li uso per home assistant o tramite App o Assistente vocale, mentre per me dovrebbe funzionare con entrambe le modalità.
Ma forse mi sono espresso male io, non so.

**Reply by Scialla** [reply]
Mi sa che hai le idee un pò confuse... Devi seguire la guida con Haaska per Alexa e per esporre i device di HomeAssistant ad Alexa.
I sensori, una volta associati alla ConBee, non li puoi associare ad altro e NON li puoi vedere sull'app Xiaomi, si fa come ho indicato sopra, cerca la guida..
